client/state/action-types.js

```diff
@@ -487,6 +487,7 @@ export const READER_FEED_SEARCH_RECEIVE = 'READER_FEED_SEARCH_RECEIVE';
 export const READER_FEED_SEARCH_REQUEST = 'READER_FEED_SEARCH_REQUEST';
 export const READER_FEED_UPDATE = 'READER_FEED_UPDATE';
 export const READER_FOLLOW = 'READER_FOLLOW';
+export const READER_FOLLOW_ERROR = 'READER_FOLLOW_ERROR';
```

Comment:
I think it would be best to not call it a READER_FOLLOW_ERROR if the only thing we mean is `READER_INVALID_FEED`.